tools/sadguess

```diff
@@ -34,19 +36,18 @@ done
 for BAS in "${ALLBAS[@]}";do
 	
 	basname=${basmap[$BAS]}
-	dirname="$BASISDIR/$BAS.sadinput"
+	dirname="$CURDIR/$BAS.sadinput"
 
 	if [  -d "$dirname" ];then
 		rm -r "$dirname"
 	fi
 	mkdir -p $dirname
 
 	cd $dirname
```

Comment:
Can this work? You go into a directory within the loop but you never leave the directory. You need a `cd ..` somewhere. But this is quite error prone. A safer way to do this is to use pushd and popd, e.g.
```
pushd $dirname
... do stuff ...
popd
```

Reply:
This worked since at line 73 there's `cd $WORKDIR`. That's why I put `WORKDIR=$QUICK_INSTALL/tools` at the top. It has been switched `pushd` and `popd` in the new commit.